pyproject.toml

```diff
@@ -43,29 +58,101 @@ source = [
    ".tox/pypy*/site-packages",
 ]
 
-[tool.black]
-skip-magic-trailing-comma = true
+[tool.pytest.ini_options]
+minversion = "8.2"
+asyncio_mode = "auto"
 
-[tool.isort]
-profile = "black"
-known_first_party = ["aiofiles"]
+[tool.ruff]
+exclude = [
+    ".direnv",
+    ".eggs",
+    ".git",
+    ".git-rewrite",
+    ".mypy_cache",
+    ".nox",
+    ".pyenv",
+    ".pytest_cache",
+    ".ruff_cache",
+    ".tox",
+    ".venv",
+    ".vscode",
+    "__pypackages__",
+    "_build",
+    "build",
+    "dist",
+    "site-packages",
+    "venv",
+]
+indent-width = 4
+line-length = 88
+target-version = "py39"
 
-[dependency-groups]
-test = [
-    "pytest>=8.2.2",
-    "pytest-asyncio>=0.23.7",
-    "coverage>=6.4.4",
-    "tox>=3.25.1",
+[tool.ruff.format]
+docstring-code-format = false
+docstring-code-line-length = "dynamic"
+indent-style = "space"
+line-ending = "auto"
+quote-style = "double"
+skip-magic-trailing-comma = false
+
+[tool.ruff.lint]
+select = [
+    # "A",  # flake8-builtins (A)
+    # "ANN",  # flake8-annotations (ANN)
+    # "ARG",  # flake8-unused-arguments (ARG)
+    "ASYNC",  # flake8-async (ASYNC)
+    "B",  # flake8-bugbear (B)
+    "BLE",  # flake8-blind-except (BLE)
+    "C4",  # flake8-comprehensions (C4)
+    # "COM",  # flake8-commas (COM)
+    # "D",  # pydocstyle (D)
+    "E",  # Error (E)
+    "EM",  # flake8-errmsg (EM)
+    "ERA",  # eradicate (ERA)
+    "F",  # Pyflakes (F)
+    # "FBT",  # flake8-boolean-trap (FBT)
+    "I",  # isort (I)
+    "ICN",  # flake8-import-conventions (ICN)
+    "ISC",  # flake8-implicit-str-concat (ISC)
+    # "N",  # pep8-naming (N)
+    "PIE",  # flake8-pie (PIE)
+    "PLE",  # Error (PLE)
+    # "PLR",  # Refactor (PLR)
+    "PLW",  # Warning (PLW)
+    # "PT",  # flake8-pytest-style (PT)
+    "PTH",  # flake8-use-pathlib (PTH)
+    "PYI",  # flake8-pyi (PYI)
+    "Q",  # flake8-quotes (Q)
+    "RET",  # flake8-return (RET)
+    "RSE",  # flake8-raise (RSE)
+    "S",  # flake8-bandit (S)
+    "SIM",  # flake8-simplify (SIM)
+    "T10",  # flake8-debugger (T10)
+    "T20",  # flake8-print (T20)
+    "TC",  # flake8-type-checking (TC)
+    # "TID",  # flake8-tidy-imports (TID)
+    "TRY",  # tryceratops (TRY)
+    "UP",  # pyupgrade (UP)
+    "W",  # Warning (W)
+    "YTT"  # flake8-2020 (YTT)
 ]
-lint = [
-    "black>=22.8.0",
-    "flake8>=7.1.1",
-    "isort==5.11.5",
+ignore = [
+    "COM812",
+    "ISC001",
+]
+fixable = [
+    "COM",
+    "I"
 ]
 
-[tool.pytest.ini_options]
-asyncio_mode = "auto"
-
-[tool.hatch.version]
```

Comment:
This is required for the release process I believe.

Reply:
It is here: https://github.com/Tinche/aiofiles/pull/216/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R46 .